lib/src/screens/restore/wallet_restore_from_keys_form.dart

```diff
@@ -54,36 +61,44 @@ class WalletRestoreFromKeysFromState extends State<WalletRestoreFromKeysFrom> {
         child: Form(
           key: formKey,
           child: Column(children: <Widget>[
-            BaseTextFormField(
-              controller: nameTextEditingController,
-              hintText: S.of(context).wallet_name,
-              validator: WalletNameValidator(),
-              suffixIcon: Container(
-                width: 12,
-                height: 14,
-                margin: const EdgeInsets.only(bottom: 15, left: 13),
-                child: InkWell(
-                  onTap: () async {
+            Stack(
+              alignment: Alignment.centerRight,
+              children: [
+                BaseTextFormField(
+                  controller: nameTextEditingController,
+                  hintText: S.of(context).wallet_name,
+                  validator: WalletNameValidator(),
+                ),
+                IconButton(
+                  onPressed: () async {
                     final rName = await generateName();
+                    FocusManager.instance.primaryFocus?.unfocus();
```

Comment:
is necessary that here ?

Reply:
@mkyq , I have tested this again by commenting this statement ` FocusManager.instance.primaryFocus?.unfocus();` and this is essential. Without this the keyboard is not dismissible.